src/Components/NavBar/Menus/UserMenu.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import { AnalyticsSchema, SystemContext } from "Artsy"
 import { useTracking } from "Artsy/Analytics/useTracking"
 import { data as sd } from "sharify"
-import { userIsAdmin } from "Utils/user"
```

Comment:
@jonallured / @pepopowitz - this `userIsAdmin` function is what I was referencing during our sprint planning. Also see the change, where one can tap into a lab feature.

Reply:
Here's an [example](https://artsyproduct.atlassian.net/browse/FX-1825) of how to create a lab feature (courtesy of @anandaroop)